lib/run/index.js

```diff
@@ -364,10 +364,11 @@ module.exports = function (options, callback) {
                     console.warn(error);
                 }
 
-                // if the reporter is not loaded then we need to warn users
-                if (!(_.isFunction(Reporter) && _.isObject(emitter.reporters[reporterName]))) {
+                // warn users if reporter is not found
+                if (!Reporter) {
                     // @todo: route this via print module to respect silent flags
-                    console.warn(`newman: "${reporterName}" reporter could not be loaded.`);
+                    console.warn(`newman: "${reporterName}" reporter could not be found.`);
```

Comment:
We should leave this error as was before. Instead, bubble up error of line L:349 as that’s the true loading error.